packages/design-system/src/components/Table/TableRow.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export interface TableRowProps {
+  /**
+   * The table row contents, usually `TableCell`.
+   */
+  children?: React.ReactNode;
+  /**
+   * @hide-prop This gets set from the parent `TableHead`
+   */
+  _isTableHeadChild?: boolean;
+}
+
+type OmitProps = 'children';
+
+export const TableRow: React.FC<
+  Omit<React.ComponentPropsWithoutRef<'tr'>, OmitProps> & TableRowProps
+> = ({ children, _isTableHeadChild, ...tableRowProps }: TableRowProps) => {
+  const renderChildren = () => {
+    return React.Children.map(children, (child: React.ReactElement) => {
```

Comment:
Am I wrong, or are you here getting more specific about the allowed types in `children`, since `React.ReactNode` also includes things like strings.

Reply:
Thanks for picking this up